projects/ngrx.io/content/guide/eslint-plugin/rules/signal-store-should-prefer-protected-state.md

```diff
@@ -0,0 +1,44 @@
+# signal-store-should-prefer-protected-state
+
+A Signal Store should prefer protected state.
+
+- **Type**: suggestion
+- **Fixable**: No
+- **Suggestion**: Yes
+- **Requires type checking**: No
+- **Configurable**: No
+
+<!-- Everything above this generated, do not edit -->
+<!-- MANUAL-DOC:START -->
+
+## Rule Details
+
+This rule should ensure that state changes are only managed by the Signal Store to prevent unintended modifications and provide clear ownership of where changes occur.
```

Comment:
```suggestion
This rule ensures that state changes are only managed by the Signal Store to prevent unintended modifications and provide clear ownership of where changes occur.
```

Reply:
Will apply it together with the entire change